### ISSUE TITLE
Safe to use Minio with pre-existing data? [documentation]

### ISSUE THREAD

**Issue by gima** [minio/minio#5050]
I was unable to find documentation regarding this.

### Question
I would like to start Minio with a data directory that has not been used with Minio before. It appears to work, but I don't know if the current behavior is unintended. If it's unintended, a future update could break this.

### What should be done
For this reason it should be documented whether this is intended functionality, for peace of mind (and data).

**Comment by deekoder** [comment]
OK we will do that @gima

**Comment by harshavardhana** [comment]
> I would like to start Minio with a data directory that has not been used with Minio before. It appears to work, but I don't know if the current behavior is unintended. If it's unintended, a future update could break this.

It is safe to use and it is a fully supported functionality not a side affect @gima

**Comment by gima** [comment]
Thank you.  
I tried to do it myself, but was confused wrt/the documentation as to where I could put this. At first it seemed the first page of documentation would be good, but it also appears to be the README of the project, and something like this probably wouldn't fit the main readme of the project.

**Comment by pwdonald** [comment]
6 months ago I tried this:

1. Start a minio server docker instance and mount an existing directory with data that had never used minio before as a volume.
2. Use the minio web UI to upload new files.

But I noticed that the newly uploaded files (objects) had metadata in the minio config directory whereas the other files did not. 

This worried me so I decided to use a different data directory and the minio client to mirror the directory so that the metadata would generate.

After reading this issue it seems that there is no problem with the fact some objects in a bucket would have metadata and others would not?

Just looking for clarification.
